app/models/queue_item.rb

```diff
@@ -0,0 +1,28 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  delegate :category, to: :video # Rails helper to replace category method
+  delegate :title, to: :video, prefix: :video
+
+  # def video_title
+  #   video.title
+  # end
+  # -> replaced by delegate :title...
```

Comment:
I would recommend deleting this code. You should know what `delegate` method does by now.